docs/index.md

```diff
@@ -0,0 +1,62 @@
+# Grabble is for Graphs
+_A Tidier, Analysis-first Data Format for Networks_
+
+## What is "a `grabble`"? 
+
+The name is inspired by the R _tidyverse_'s **tibble**
+
+> _**Gra**ph Ti**bble**_
+
+As such, a `grabble` is meant to be a _tidy_ data structure (see REF), enabling a host of analysis and exploration _quality-of-life_
+improvements for network analysts. 
+
+> _a modern reimagining of **graph data structures**, keeping what time has proven to be effective, and throwing out what is not._
+
+Grabble is also a highly _general_ way of storing and manipulating graphs: it is intended as a _pivot_ format, that eases the transition from all the various formats for computationally representing graphs, while also making numerical 
```

Comment:
Did you want to add something else here?

Reply:
Yeah, I'm going to work on this for a bit on my separate fork/branch, but def. feel free to comment/ask q's as you think of them!